pims/image_sequence.py

```diff
@@ -16,6 +19,7 @@
 
 # skimage.io.plugin_order() gives a nice hierarchy of implementations of imread.
 # If skimage is not available, go down our own hard-coded hierarchy.
+from PIL import Image
```

Comment:
I always get confused with the PIL/pillow conventions.  Does this work with both?  We should prefer pillow to PIL.